backend/mora/service/employee.py

```diff
@@ -199,7 +224,10 @@ def get_one_employee(c, userid, user=None, full=False):
 
         r[mapping.ORG] = org.get_one_organisation(c, orgid)
         r[mapping.USER_KEY] = props['brugervendtnoegle']
-
+    elif details is EmployeeDetails.MINIMAL:
+        pass  # already done
+    elif details is EmployeeDetails.INTEGRATION:
+        r["integration_data"] = props.get("integrationsdata", "")
```

Comment:
... for hvis du ikke gav integrationsdata særbehandling herover, ville du ikke behøve at gøre det her.

Reply:
Samme kommentar som ovenfor: Dette er udelukkende for at brugerinterfacet ikke skal gøre noget for at undgå integrationsdata, men de, som vil anvende dem - integrationsprogrammer - skal specifikt bede om at se dem.

Reply:
Samme kommentar som ovenfor - det er ikke if-sætningen, jeg brokker mig over :slightly_smiling_face:

Reply:
Jeg har forfægtet min forsigtighed ovenfor.